docs/r2/graphs.ipynb

```diff
@@ -0,0 +1,492 @@
+        "id": "F-2yw5qd7OpK",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_computation.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "jDRynpVw53SJ",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_computation_detail.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Oj9FSPdz6SO2",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Conceptual graph\n",
+        "\n",
+        "In addition to the execution graph, TensorBoard also displays a **conceptual graph**. This is a view of just the Keras model. This may be useful if you’re reusing a saved model and you want to examine or validate its structure.\n",
+        "\n",
+        "To see the conceptual graph, select the “keras” tag. For this example, you’ll see a collapsed **Sequential** node. Double-click the node to see the model’s structure:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "Qw9rbEcE6eZB",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_tag_selection.png?raw=1\"/> <br/>\n",
+        "<img class=\"tfo-display-only-on-site\" src=\"https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/graphs_conceptual.png?raw=1\"/>"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "zVuaKBifu-qF",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Graphs of tf.functions\n",
+        "\n",
+        "The examples so far have described graphs of Keras models, where the graphs have been created by defining Keras layers.\n",
+        "\n",
+        "The [```tf.function```](https://www.tensorflow.org/r2/guide/autograph) annotation in TensorFlow 2.0 allows you to transform a Python computation function into portable, high-performance TensorFlow graphs. This is called **autographing** a function. The **TensorFlow Trace API** generates graph data for autographed functions for visualization in TensorBoard."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "JIuhJnQ8w-dT",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Here's how you use the Trace API:\n",
+        "\n",
+        "*   Define and annotate a function with `tf.function`\n",
+        "*   Use `tf.summary.trace_on()` immediately before your function call site\n",
+        "*   With a Summary file writer, call `tf.summary.trce_export()` to save the log data\n",
```

Comment:
typo in `trce`

Reply:
Fixed typo. Reworded but I'm not worrying about the transition much because this doc is about TensorBoard, not about tf.function() or autograph. I've linked to those docs and those tutorials should specify where and when the user needs to use them.